docs/overview/presentations.md

```diff
@@ -0,0 +1,27 @@
+# Presentations
```

Comment:
Is there any point breaking this out into a separate page? (visual scan - click - page reload etc)

Reply:
My initial thought was it would be a living page where we can put all the presentations done for meaningful features, workshops, conferences, etc. I wasn't sure if it was something we wanted to add or not so I figured I would get some feedback. We could even rename it to `Conferences + Meetups` or something similar. Then people will have an easy place to go to see all the places OpenFaaS has been presenting. That was my thought process here if needs more stuff added and the format adjusted slightly, for sure.

Reply:
That makes sense - right now it's contained in the community.md file in the main faas repo which is linked in the community page.

Ideally I think the initial docs PR should make minimal changes to the structure and be relatively flat. Since this page will be published as soon as the PR is merged an empty landing page is not as useful in the short-term.